index.js

```diff
@@ -1,5 +1,8 @@
 let items = [];
 let newDiv;
+const MODAL_HEADER = `<button class="close" data-dismiss="modal">x
```

Comment:
const declarations should be placed before let declarations according with lint rules